py-polars/src/lazy/dataframe.rs

```diff
@@ -839,6 +839,11 @@ impl PyLazyFrame {
         ldf.melt(args).into()
     }
 
+    pub fn approx_unique(&self) -> Self {
```

Comment:
This should be removed. (expression API)

Reply:
done